packages/flutter_markdown/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.8.0
```

Comment:
This should be 0.7.7, unless it's a breaking change, which I don't think it should be.